src/Stripe.net/Services/_base/Service.cs

```diff
@@ -258,7 +258,7 @@ protected EntityReturned CreateEntity(BaseOptions options, RequestOptions reques
                 string itemId = null;
                 foreach (var item in page)
                 {
-                    itemId = ((IHasId)item).Id;
+                    itemId = (item as IHasId)?.Id;
```

Comment:
What is this change doing?

Reply:
Makes the code a little bit safer. If for some reason `item` is not castable to `IHasId`, then `itemId` will be `null` and the iterator will stop as if it had reached the end of the list. The current version crashes if `item` cannot be cast to `IHasId`. (In practice, this should never happen: all listable resources have IDs. I think stripe-mock was returning invalid data in one of the tests but I didn't dig too deep.)

Reply:
Ack. Can we make this its own PR? sorry to be annoying but this is the kind of change that will cause us trouble because we suddenly start returning more results/things.